Add documentation for gcp-bucket module (#3994)
wazuh/wazuh-documentation: source/gcp/prerequisites/considerations.rst

Considerations for configuration
================================

First execution
---------------

If no ``only_logs_after`` value was provided, the module will only fetch the logs of the date of the execution.

Older logs
----------

The ``Google Cloud Platform`` Wazuh module only looks for new logs in buckets based upon the key of the last processed log object, which includes the datetime stamp. If older logs are loaded or the ``only_logs_after`` option date is set to a datetime earlier than previous executions of the module, the older log files will be ignored and not ingested into Wazuh.

Creation time in Google Cloud Storage bucket contents
-----------------------------------------------------

> When using the ``only_logs_after`` tag, the Wazuh module uses the creation time of each blob in the Google Cloud Storage bucket to determine if a file should be processed or not. This means that if the user manually moves any blob inside the specified bucket, its creation date changes and the ``gcp-module`` Wazuh module processes it again as it is considered a new blob.

Author: ```suggestion
When using the ``only_logs_after`` tag, the Wazuh module checks the creation time of each blob in the Google Cloud Storage bucket to determine if a file should be processed or not. This means that if the user manually moves any blob inside the specified bucket, its creation date changes and the ``gcp-module`` Wazuh module processes it again as it is considered a new blob.
```
```suggestion
When using the ``only_logs_after`` tag, the Wazuh module uses the creation time of each blob in the Google Cloud Storage bucket to determine if a file should be processed or not. This means that if the user manually moves any blob inside the specified bucket, its creation date changes and the ``gcp-module`` Wazuh module processes it again as it is considered a new blob.
```

> When using the ``only_logs_after`` tag, the Wazuh module uses the creation time of each blob in the Google Cloud Storage bucket to determine if a file should be processed or not. This means that if the user manually moves any blob inside the specified bucket, its creation date changes and the ``gcp-module`` Wazuh module processes it again as it is considered a new blob.

Author: ```suggestion
When using the ``only_logs_after`` tag, the Wazuh module checks the creation time of each blob in the Google Cloud Storage bucket to determine if a file should be processed or not. This means that if the user manually moves any blob inside the specified bucket, its creation date changes and the ``gcp-module`` Wazuh module processes it again as it is considered a new blob.
```

----------

The ``gcp-pubsub`` Wazuh module only looks for new logs based upon the key for last processed log object, which includes the datetime stamp. If older logs are loaded into the Cloud Storage or the ``only_logs_after`` option date is set to a datetime earlier than previous executions of the module, the older log files will be ignored and not ingested into Wazuh.

Creation time in Google Cloud Storage bucket contents
-----------------------------------------------------

> The ``gcp-bucket`` Wazuh module will use the creation time for each blob inside the provided Cloud Storage bucket to determine the date to which each log file corresponds. This means if the user manually move any file inside the specified bucket its creation time will be updated and the ``gcp-module`` will assume the file is new.

Reviewer: ```suggestion
The ``gcp-bucket`` Wazuh module will use the creation time for each blob inside the provided Cloud Storage bucket to determine the date to which each log file corresponds. This means if the user manually moves any file inside the specified bucket its creation time will be updated and the ``gcp-module`` will assume the file is new.
```

> The ``gcp-bucket`` Wazuh module will use the creation time for each blob inside the provided Cloud Storage bucket to determine the date to which each log file corresponds. This means if the user manually moves any file inside the specified bucket its creation time will be updated and the ``gcp-module`` will assume the file is new.

Reviewer 2: ```suggestion
When using the ``only_logs_after`` tag, the Wazuh module uses the creation time of each blob in the Google Cloud Storage Bucket to determine if a file should be processed or not. This means that if the user manually moves any blob inside the specified bucket, its creation date changes and the ``gcp-module`` Wazuh module processes it again as it is considered a new blob.

```
